src/components/FormLayout/__tests__/FormLayout.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import type { HTMLAttributes } from 'react';
+import {
+  FormLayout,
+  FormLayoutCustomField,
+} from '..';
+import { TextField } from '../../TextField';
+
+// Types for story component will be improved when we have full TypeScript support
+type FormLayoutForTestProps = HTMLAttributes<HTMLDivElement>;
+type FormLayoutWithCustomFieldForTestProps = FormLayoutForTestProps & {
+  customFieldLayoutProps: HTMLAttributes<HTMLDivElement>;
+  customFieldProps: HTMLAttributes<HTMLDivElement>;
+};
+
+export const FormLayoutForTest = ({
+  ...props
+}: FormLayoutForTestProps) => (
+  <FormLayout {...props}>
+    <TextField label="Label1" />
```

Comment:
There should be all the input components + FormLayoutCustomField so we know it works with all the inputs as expected.